package.json

```diff
@@ -24,7 +24,7 @@
     "test:validator:integrity": "npm run build:validation && git diff --quiet --ignore-all-space --ignore-blank-lines --ignore-cr-at-eol lib/error-serializer.js && git diff --quiet --ignore-all-space --ignore-blank-lines --ignore-cr-at-eol lib/configValidator.js",
     "test:typescript": "tsc test/types/import.ts --noEmit && tsd",
     "test:watch": "npm run unit -- --watch --coverage-report=none --reporter=terse",
-    "unit": "tap",
+    "unit": "borp --coverage --check-coverage --lines 100 --reporter ./test/test-reporter.mjs",
     "unit:junit": "tap-mocha-reporter xunit < out.tap > test/junit-testresults.xml",
     "unit:report": "tap --coverage-report=html --coverage-report=cobertura | tee out.tap",
```

Comment:
We need to replace all instances of using `tap` to run tests. There's still this one, and at least one more on line 30.